.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
you can add this folder in .gitignore file , so they don't get committed, 
these are just your vs code settings.

Reply:
we look into it ,  Thanks